pkgs/joinmarket/get-sha256.sh

```diff
@@ -1,25 +1,22 @@
-#!/usr/bin/env bash
-set -euo pipefail
-. "${BASH_SOURCE[0]%/*}/../../helper/run-in-nix-env" "git gnupg" "$@"
-
-TMPDIR="$(mktemp -d -p /tmp)"
-trap 'rm -rf $TMPDIR' EXIT
-cd "$TMPDIR"
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p git gnupg jq
 
-echo "Fetching latest release"
-git clone https://github.com/joinmarket-org/joinmarket-clientserver 2> /dev/null
-cd joinmarket-clientserver
-latest=$(git describe --tags "$(git rev-list --tags --max-count=1)")
-echo "Latest release is $latest"
+set -euo pipefail
+newVersion=$(curl -s "https://api.github.com/repos/joinmarket-org/joinmarket-clientserver/releases" | jq -r '.[0].tag_name')
 
-# GPG verification
-export GNUPGHOME=$TMPDIR
-echo "Fetching Adam Gibson's key"
```

Comment:
Can we get this line back in? This adds a bit of context to the bare pubkey.

Reply:
Done